src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
```

Comment:
The function name `makeInfinityAdder` does not match the task requirements. It should be `makeAdder` according to the task description.